static/css/inbox.less

```diff
@@ -245,6 +245,15 @@ body {
   }
 }
 
+.sync-status {
+  .in_progress, .not_required {
+    color: green;
```

Comment:
Always try and use existing variables for colours because then they'll be consistent with our palette rather than every commit adding a new colour. Also if we decide to change what colour should represent "ok" then we only need to change it in one place. The current variables are defined in variables.less. Consider using `@ok-color`.